python/nav/jwtconf.py

```diff
@@ -32,7 +32,7 @@ def get_issuers_setting(self):
             configparser.NoOptionError,
             ConfigurationError,
         ) as error:
-            _logger.error('Error reading jwtconfig %s', error)
+            _logger.error('Error reading jwtconfig {}'.format(error))
```

Comment:
No, don't use `.format()` for log strings. The log handler will format strings itself using the syntax you replaced. Using the internal formatting of the logger ensures it doesn't waste time formatting things into strings if the configured log level prevents this log message from being emitted

Reply:
There is a way to use f-strings instead of old-style interpolation in logs but it is a hassle to configure and we don't do it anywhere else, so let's stick to old-style.